src/main/java/seedu/duke/Duke.java

```diff
@@ -24,8 +19,21 @@ private static void bye() {
     }
 
     public static void main(String[] args) throws NullException {
+        UserInterface ui;
+        // boolean isSaveFileLoaded = hasSaveFileLoad();
+
         hello();
-        UserInterface.run();
+        if (false) {
+            // load save file;
+            // int capacity = 0;
+            // welcome user back;
+            // ui = new UserInterface(new Warehouse(capacity));
+        } else {
+            System.out.println("New login. Please type the total number of goods"
+                    + " your warehouse can hold");
```

Comment:
pseudocode for how i think loading a file in will look w.r.t. how to initialise the warehouse capacity